lib/util.js

```diff
@@ -88,6 +86,24 @@ async function validateWebhook(requestData, secret) {
   );
 }
 
+/**
+ * @param {string} secret - base64 encoded string
+ * @param {string} data - text body of request
+ */
+async function createHMACSHA256(secret, data) {
+  const encoder = new TextEncoder();
+  const key = await crypto.subtle.importKey(
```

Comment:
Do you still need the global `import crypto from 'crypto'` in order to run this code? The test failures would suggest that a global `crypto` isn't available, but I'm not sure whether that's an environmental thing or specific to CI.

Reply:
Nope, weird, it's available as a global in node 18, I wonder if this is a ts-jest issue… will look into it.